test/utils.cpp

```diff
@@ -648,6 +648,34 @@ TEST_CASE("getcwd() returns current directory of the process", "[utils]")
 	}
 }
 
+TEST_CASE("strnaturalcmp() compares strings appropriately", "[utils]")
```

Comment:
I understand that "natural" already hints at what is "appropriate" for this function, but let's be explicit and say that the function "compares strings using natural numeric ordering" or something.